austation/code/modules/mining/machine_vending.dm

```diff
@@ -0,0 +1,5 @@
+/obj/machinery/mineral/equipment_vendor/Initialize()
+	..()
+	for(var/datum/data/mining_equipment/R in prize_list)
+		if ((R.name == "Minebot Upgrade: A.I.") || (R.name == "Mining Bot Companion"))
+			prize_list.Cut(R)
```

Comment:
this *should* work, but I'd rather a list of items to remove.
you can do this by adding a variable with the list to the mining vendor object, and then doing
`R.name in austation_remove_prize_list` instead of writing out the whole check like that.

Reply:
Will do tonight

Reply:
done